test/black-box/ListingItem.test.ts

```diff
@@ -267,6 +267,53 @@ describe('/listing-items', () => {
 
     });
 
+
+
```

Comment:
It's good that you're testing the rpc endpoint. This test class is auto generated for the rest api and since we don't actually use the rest api for anything, could you create a separate test file for this? We should perhaps have a separate test file for each of our rpc commands or group of similar commands, so maybe move this to RpcGetItem.test.ts?